extensions/checksum/README.md

```diff
@@ -2,30 +2,29 @@
 
 **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
 
-Provides a way to specify MD5, SHA1, SHA2 and SHA3 file checksums for assets and links in STAC Items, STAC Catalogs and STAC Collections.
+Provides a way to specify file checksums (e.g. BLAKE2, MD5, SHA1, SHA2, SHA3) for assets and links in STAC Items, STAC Catalogs and STAC Collections. The hashes are self-identifying hashes as described in the [Multihash specification](https://github.com/multiformats/multihash).
 
-- [Example](examples/example-sentinel1.json)
-- [JSON Schema](json-scehma/schema.json)
+- [Example](examples/sentinel1.json)
+- [JSON Schema](json-schema/schema.json)
 
 ## [`Link Object`](../../item-spec/item-spec.md#link-object) and [`Asset Object`](../../item-spec/item-spec.md#asset-object) fields
 
-| Field Name    | Type   | Description                                                  |
-| ------------- | ------ | ------------------------------------------------------------ |
-| checksum:md5  | string | [MD5 message-digest algorithm](https://en.wikipedia.org/wiki/MD5) 128-bit checksum for the corresponding file, encoded as hexadecimal string. |
-| checksum:sha1 | string | [SHA-1 (Secure Hash Algorithm 1)](https://en.wikipedia.org/wiki/SHA-1) 160-bit checksum for the corresponding file, encoded as hexadecimal string. |
-| checksum:sha2 | string | [SHA-2 (Secure Hash Algorithm 2)](https://en.wikipedia.org/wiki/SHA-2) checksum for the corresponding file with varying sizes of 224, 256, 384 or 512 bits, encoded as hexadecimal string. |
-| checksum:sha3 | string | [SHA-3 (Secure Hash Algorithm 3)](https://en.wikipedia.org/wiki/SHA-3) checksum for the corresponding file with varying bit sizes, encoded as hexadecimal string. |
+| Field Name | Type   | Description                                                  |
+| ---------- | ------ | ------------------------------------------------------------ |
+| checksum   | string | Multihash for the corresponding file, encoded as hexadecimal (base 16) string with lowercase letters. |
```

Comment:
I think I'd prefer the field name to be `checksum:multihash` for three reasons:
1. We have a prefix (checksum) as for all other extensions.
2. We could extend it later (e.g. having checksum:newhash)
3. From the field name it is clear how the hash is "named" without needing to read the docs.